_drafts/oryx/technology.md

```diff
@@ -0,0 +1,60 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+When possible and appropriate, Oryx adheres to standards for both technologies and tools.
+Oryx strives to minimize the use of build tools and instead utilize the native capabilities of the web, as much as possible.
+Oryx aims to use JavaScript during development, but tries avoid the need to run JavaScript in the browser. Wherever possible, HTML and CSS are preferred to render a fast loading experience cross different screen sizes.
+
+We will examine both the deployed technology stack and the technologies utilized for development in order to gain a more comprehensive understanding of the technologies.
+
+## Deployed stack:
+
+### HTML & CSS & JS
+HTML, CSS, and JS are considered to be the fundamental and essential set of web technologies used for creating web pages. HTML is responsible for the structure and content of the page, CSS is used for styling and layout, and JS adds interactivity and dynamic behavior to the page. Together, these three technologies form the foundation of modern web development.
```

Comment:
It would actually be nice to express that:
- we try to avoid loading JS in the client unless there's a need; we gradually load snippets of JS rather than a full JS framework
- we have a CSS first approach. CSS is always preferred to render the styles and layouts, as it won't requires JS logic.